test/model.test.ts

```diff
@@ -203,6 +204,7 @@ describe(OCX.Model, () => {
                   numerator: opts.to,
                   denominator: opts.convertsFrom,
                 },
+                rounding_type: rounding_type,
```

Comment:
You changed a test helper function but you never call it with this parameter so, why?

**Please fix:** Please write tests for the model.